packages/@magic-sdk/react-native-bare/README.md

```diff
@@ -0,0 +1,51 @@
+# ✨ Magic Authentication JavaScript SDK
+
+[![<MagicLabs>](https://circleci.com/gh/magiclabs/magic-js.svg?style=shield)](https://circleci.com/gh/magiclabs/magic-js)
```

Comment:
Could we state in the README.md that this is a package split and everything is in beta?
"If anything doesn't work, you may open an issue and revert the package to the previous stable version "magic-sdk/react-native@x.x.x""